tests-utils/src/main/java/tech/beshu/ror/utils/elasticsearch/SearchManagerJ.java

```diff
@@ -51,18 +51,33 @@ public SearchResult search(String endpoint) {
     return call(createSearchRequest(endpoint), SearchResult::new);
   }
 
+  public SearchResult storeScript(String scriptId, String query) { return call(createStoreScriptRequestRequest(scriptId, query), SearchResult::new); }
```

Comment:
do we need this method here? I see you've created ScriptManager?

Reply:
Nope.